### ISSUE TITLE
"bundle format unrecognized ..."

### ISSUE THREAD

**Issue by ghost** [alawong/react-native-braintree#24]
After the pod install, it get two errors on build: .. /Braintree-UI-Localization.bundle: bundle format unrecognized, invalid, or unsuitable
 and .. /Braintree-Drop-In-Localization.bundle: bundle format unrecognized, invalid, or unsuitable

Anyone else experience this? Any ideas?

**Comment by lewisbarber** [comment]
Yep, I'm getting this as well. It's complaining about a code signing issue, and I have no idea why!

<img width="401" alt="screen shot 2016-12-08 at 11 15 15" src="https://cloud.githubusercontent.com/assets/7353559/21008019/adf34a5a-bd37-11e6-80f7-f7d21e51fc9f.png">
<img width="795" alt="screen shot 2016-12-08 at 11 15 22" src="https://cloud.githubusercontent.com/assets/7353559/21008018/adf2600e-bd37-11e6-9dce-631493124b35.png">

Is this the same error you're getting? Did you manage to solve it?

**Comment by lewisbarber** [comment]
The answer by **Scriptable** fixed it for me: http://stackoverflow.com/questions/29271548/code-sign-error-bundle-format-unrecognized-invalid-or-unsuitable
